### ISSUE TITLE
utf-8 decode issue

### ISSUE THREAD

**Issue by FeelsBadMan1** [onemanbuilds/AuthGGDCBot#1]
Hello, i ran the code, edited the config with my apikey and so on and not changed anything on the code. I then ran the .usercount command as a test but this issue also happen on every other command i run. Here is the problem:

![grafik](https://user-images.githubusercontent.com/40109798/132962325-20c740a6-04f4-4852-b3aa-524ba5e3e2d2.png)

I managed to get where the error happend and what caused it. seems like its this line within the configs.json file:
``
"authkey":"authkeychangeitwithtebot"
``
Im not sure what exactly i hsould put in there. everything else i filled with my stuff

**Comment by onemanbuilds** [comment]
Sorry i forgot to mention this in the documentation you have to set up the bot with the commands not manually in the configs.json because it will encrypt the auth keys and so on to make sure it wont be leaked or something like that, if i remember right this can be the issue.

So set up the owner id in the configs.json manually and then use the owner commands to set up the aid and so on.

**Comment by FeelsBadMan1** [comment]
oh okay. Thank u for the anser; i will try this solution when im at home.

**Comment by FeelsBadMan1** [comment]
> 
> 
> Sorry i forgot to mention this in the documentation you have to set up the bot with the commands not manually in the configs.json because it will encrypt the auth keys and so on to make sure it wont be leaked or something like that, if i remember right this can be the issue.
> 
> So set up the owner id in the configs.json manually and then use the owner commands to set up the aid and so on.

Oh okay but the encryption using base64 wont help with someone not leaking it. there are lots of websites where you just put in the encoded string and it will decode within miliseconds. You also wont give the discordbot files to someone else and host on a linux machine.

Edit:
Now the setup worked but every command i run seems to fail....
I am also not quite sure where to get the authkey
![grafik](https://user-images.githubusercontent.com/40109798/133810465-5bf64c62-7b0a-4790-ae80-91345861198a.png)

**Comment by onemanbuilds** [comment]
Yeah thats true i have a tool for that too, thats why i wanted to add a custom salt option for the encryption. For most of the functions you will need a valid auth.gg subscription there is an application secret which you can find at the applications tab, and if i remember right you can find the rest at the account settings page.

**Comment by FeelsBadMan1** [comment]
ah okay. well i have a premium account on authgg so this is not a issue. On the Application on auth.gg is a appsecret but also a aid so im not sure what aid i need to use and so on. also there is nothing called authkey on the website so idk what to enter on .setauthkey
